lisa/transformers/dom0_kernel_installer.py

```diff
@@ -80,6 +81,9 @@ def install(self) -> str:
         uname = node.tools[Uname]
         current_kernel = uname.get_linux_information().kernel_version_raw
 
+        mariner_version = node.os.information.version
+        print(f"mariner_version: {mariner_version}")
```

Comment:
Don't use print in code. The version number should be in debug information already, it doesn't need to be printed again.

Reply:
It was just temporary print. I removed it in last push.